pkg/api/helpers.go

```diff
@@ -12,6 +12,7 @@ import (
 	"sync"
 	"time"
 
+	"github.com/moul/anonuuid"
```

Comment:
Do not forget the "party -c -t -d=vendor" separated commit when you add new imports